dev/com.ibm.websphere.appserver.features/visibility/public/jaxrsClient-3.0/com.ibm.websphere.appserver.jaxrsClient-3.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxrsClient-3.0
+visibility=public
+singleton=true
+IBM-API-Package: javax.ws.rs; type="spec", \
+ javax.ws.rs.container; type="spec", \
+ javax.ws.rs.core; type="spec", \
+ javax.ws.rs.client; type="spec", \
+ javax.ws.rs.ext; type="spec", \
+ javax.ws.rs.sse; type="spec", \
+ com.ibm.websphere.jaxrs20.multipart; type="ibm-api", \
+ com.ibm.websphere.jaxrs.providers.json4j; type="ibm-api"
```

Comment:
@WhiteCat22 can you open an issue (and reference the Epic Issue #11803) to determine what we should do with providers.json4j (I'd like to remove it) and jaxrs20.multipart (this will likely need some changes as the multipart support is based on CXF's implementation and might not work unchanged with RESTEasy).  I'm fine to push this in as-is since this is currently undecided, but we should not lose track of these things.  Thanks.